cloud/gcp-functions/cmd/orphanedNeg.go

```diff
@@ -50,16 +50,16 @@ func CleanOrphanedNEG(w http.ResponseWriter, r *http.Request) {
 					break
 				}
 				if err != nil {
-					log.Fatalf("Failed to list NEGs: %v", err)
+					log.Fatal("Failed to list NEGs: %v", err)
 				}
 
-				log.Printf("Checking NEG: %s\n", neg.GetName())
+				log.Print("Checking NEG: %s\n", neg.GetName())
```

Comment:
and you need to use `Printf` here.

log messages have newline by default, so you just need to remove `\n` at the end.